app/src/main/java/com/framgia/musixmatch/data/model/Album.java

```diff
@@ -0,0 +1,89 @@
+package com.framgia.musixmatch.data.model;
+
+public class Album {
+    private int mAlbumId;
+    private String mAlbumName;
+    private int mArtistId;
+    private String mArtistName;
+    private int mNumberOfSongs;
+    private String mAlbumArt;
+
+    public Album(AlbumBuilder albumBuilder) {
+        mAlbumId = albumBuilder.mAlbumId;
+        mAlbumName = albumBuilder.mAlbumName;
+        mArtistId = albumBuilder.mArtistId;
+        mArtistName = albumBuilder.mArtistName;
+        mNumberOfSongs = albumBuilder.mNumberOfSongs;
+        mAlbumArt = albumBuilder.mAlbumArt;
+    }
+
+    public int getAlbumId() {
+        return mAlbumId;
+    }
+
+    public String getAlbumName() {
+        return mAlbumName;
+    }
+
+    public int getArtistId() {
+        return mArtistId;
+    }
+
+    public String getArtistName() {
+        return mArtistName;
+    }
+
+    public int getNumberOfSongs() {
+        return mNumberOfSongs;
+    }
+
+    public String getAlbumArt() {
+        return mAlbumArt;
+    }
+
+    public static class AlbumBuilder {
```

Comment:
```suggestion
    public static class Builder {
```
e đặt tên ntn là đủ vì nó là nested class nên khi gọi sẽ là Album.Builder